src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/Tov.cpp

```diff
@@ -220,17 +216,15 @@ void TovSolution::integrate(
       // Transform observed radius to isotropic, so we use the isotropic radius
       // for all interpolations below
       observer.radius[i] /= square(observer.conformal_factor[i]);
-      // The interpolation is not safe otherwise
     }
+    // The interpolation is not safe otherwise
     observer.radius.back() = outer_radius_;
```

Comment:
@isaaclegred what goes wrong with the interpolation if you don't do this?

Reply:
Aren't you doing extrapolation if you don't do this? That's naively what I would be concerned about. You want the interpolation to be `[0, outer_radius]`

Reply:
Floating point error in the line above can make the last `observer_radius` value go above the `outer_radius` value, so if you want to interpolate to `outer_radius` you have to explicitly include it.